fortinet-fortirecon-brand-protection/release_notes.md

```diff
@@ -1,25 +1,15 @@
-#### What's Fixed in 2.0.0
+#### What's Fixed in 2.1.0
```

Comment:
please update the release notes to:
#### What's Fixed in 2.1.0
- Added the following actions:
  - Get Code Repositories
  - Get Code Repos Statistics
  - Get Code Repos Matched Domains Statistics
- In 'Get Rogue Apps' action updated the options for status and added the following parameters: App Store Name, Start Date, End Date.
- In 'Get Takedown Requests' action updated the options for status and renamed the 'Keyword' parameter to 'Search By Keyword'.
- Updated the output schema for following actions: Get Rogue Apps, Get Rogue App By ID, Get Takedown Requests.
- Deprecated the following actions:
  - Get Code Repo Exposures Statistics
  - Get Matched Domains Statistics

Reply:
Done